src/SSHDebugPS/VsOutputWindowWrapper.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+using System;
+using System.Collections.Generic;
+
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.SSHDebugPS
+{
+    internal static class VsOutputWindowWrapper
+    {
+        private static Lazy<IVsOutputWindow> outputWindowLazy = new Lazy<IVsOutputWindow>(() =>
+        {
+            return ThreadHelper.JoinableTaskFactory.Run(async delegate
+            {
+                IVsOutputWindow outputWindow = null;
+                try
+                {
+                    await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
+                    outputWindow = Package.GetGlobalService(typeof(SVsOutputWindow)) as IVsOutputWindow;
+                }
+                catch (Exception)
+                {
+                }
+                return outputWindow;
+            });
+        });
+        private static Lazy<IVsUIShell> shellLazy = new Lazy<IVsUIShell>(() =>
+        {
+            return ThreadHelper.JoinableTaskFactory.Run(async delegate
+            {
+                IVsUIShell shell = null;
+                try
+                {
+                    await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
+                    shell = Package.GetGlobalService(typeof(SVsUIShell)) as IVsUIShell;
+                }
+                catch (Exception)
+                {
+                }
+                return shell;
+            });
+        });
+
+
+        private class PaneInfo
+        {
+            public PaneInfo(Guid paneId)
+            {
+                this.paneId = paneId;
+                this.Shown = false;
+            }
+
+            internal Guid paneId;
+            internal bool Shown { get; set; }
+        }
+
+        private static Dictionary<string, PaneInfo> panes = new Dictionary<string, PaneInfo>()
+        {
+            // The 'Debug' pane exists by default
+            { "Debug", new PaneInfo(VSConstants.GUID_OutWindowDebugPane) }
```

Comment:
i feel like with the same string being used in 3 places (at least) you should make a const  and refer to it.